src/client/app/saksbehandler/behandlingskoer/components/OppgavekoPanel.tsx

```diff
@@ -46,33 +49,49 @@ const OppgavekoPanel: FunctionComponent<OwnProps> = ({
 }) => {
   const [visBehandlingerIKo, setVisBehandlingerIKo] = useState<boolean>(false);
   const [visReservasjoneriKo, setVisReservasjonerIKO] = useState<boolean>(true);
+  const [visHastesakReservasjoner, setVisHastesakReservasjoner] = useState<boolean>(true);
   const [visFinnesIngenBehandlingerIKoModal, setVisFinnesIngenBehandlingerIKoModal] = useState<boolean>(false);
   const {
-    startRequest: fåOppgaveFraKo, state: restApiState, error: restApiError, resetRequestData,
+    startRequest: fåOppgaveFraKo,
+    state: restApiState,
+    error: restApiError,
+    resetRequestData,
   } = useRestApiRunner<Oppgave>(K9LosApiKeys.FÅ_OPPGAVE_FRA_KO);
+  const merknadstyper = useKodeverk(kodeverkTyper.MERKNAD_TYPE);
 
-  const valgtKo = oppgavekoer.find((ko) => ko.id === valgtOppgavekoId);
+  const valgtKo = oppgavekoer.find(ko => ko.id === valgtOppgavekoId);
 
   useEffect(() => {
-    if (restApiState && restApiState === RestApiState.ERROR && restApiError && restApiError.toString().includes('404')) {
+    if (
+      restApiState &&
+      restApiState === RestApiState.ERROR &&
+      restApiError &&
+      restApiError.toString().includes('404')
+    ) {
       setVisFinnesIngenBehandlingerIKoModal(true);
       resetRequestData();
     }
   }, [restApiState, restApiError]);
 
   const plukkNyOppgave = () => {
-    fåOppgaveFraKo({ oppgaveKøId: valgtOppgavekoId }).then((reservertOppgave) => {
+    fåOppgaveFraKo({ oppgaveKøId: valgtOppgavekoId }).then(reservertOppgave => {
       resetRequestData();
       apneOppgave(reservertOppgave);
     });
   };
 
-  const sorterteOppgavekoerIBokstavsordning = oppgavekoer.sort((a, b) => a.navn.localeCompare(b.navn));
+  const hastesaker = reserverteOppgaver.filter(oppgave => oppgave.merknad?.merknadKoder?.includes('HASTESAK'));
```

Comment:
bruke kodeverk her istedenfor 'HASTESAK' kanskje?

Reply:
har du noe forslag til hvordan det kan gjøres? For å bruke koden fra MerknadType i kodeverk må jeg kunne finne riktig objekt i MerknadType-listen. Da må jeg bruke HASTESAK-stringen for å finne riktig objekt, og da ender jeg jo bare opp med å bruke en string til å hente ut samme string fra kodeverk

Reply:
ja sant... Du har ikke frontend-kodeverk som kloner MerknadType som du kan evt bruke?

Reply:
legger til merknadType.ts